src/blocks/FormCard.scss

```diff
@@ -87,11 +88,16 @@
 }
 
 .form-card__group {
-  @apply px-8;
+  @screen md {
+    @apply px-8;
+  }
+
+  @screen print {
```

Comment:
More out of curiosity, but what does "print" mean here?

Reply:
To me it looks like a holdover that requires cleanup --> seeds, was just worried about messing with it here